### ISSUE TITLE
readNumber(Token.UINT64_LE) throws error

### ISSUE THREAD

**Issue by Borewit** [Borewit/strtok3#80]
Will throw error:
```
(node:7253) UnhandledPromiseRejectionWarning: RangeError [ERR_OUT_OF_RANGE]: The value of "length" is out of range. It must be >= 0 && <= 4. Received 8
```

**Comment by Borewit** [comment]
Workaround: use:
```js
const x = await tokenizer.readToken(Token.UINT64_LE);
```

**Comment by Borewit** [comment]
Fixed by #82, in [v3.1.3](https://github.com/Borewit/strtok3/releases/tag/v3.1.3)
